feature-libs/qualtrics/components/qualtrics-loader/qualtrics-loader.service.ts

```diff
@@ -38,8 +45,8 @@ export class QualtricsLoaderService implements OnDestroy {
   /**
    * QSI load event that happens when the QSI JS file is loaded.
    */
-  private qsiLoaded$: Observable<any> = this.winRef?.nativeWindow
-    ? fromEvent(this.winRef.nativeWindow, QUALTRICS_EVENT_NAME)
+  private qsiLoaded$: Observable<any> = isPlatformBrowser(this.platformId)
+    ? fromEvent(this.window as QualtricsWindow, QUALTRICS_EVENT_NAME)
```

Comment:
You don't need to cast it to `QualtricsWindow` here, as the `this.window` already returns it.
You most likely have to check if the `this.window` exists, and you can do it like this: `isPlatformBrowser(this.platformId) && this.window`